src/howtos/install_epinio_on_rancher.md

```diff
@@ -4,28 +4,53 @@ Before installing Epinio, you need a running Rancher instance.
 
 If you do not have such, please use the Rancher's [installation guides](https://rancher.com/docs) to set one up.
 
-When I wrote this documentation, I used the following versions:
-* epinio-installer (Helm chart) 0.3.5 
+This page was written using the following versions:
+* [epinio helm chart 0.7.1](https://github.com/epinio/helm-charts/releases/tag/epinio-0.7.1)
 * Rancher 2.6.3
 
-## Epinio Prerequisites
+## Kubernetes cluster
 
-### Kubernetes cluster
 The two supported methods are to either deploy a new cluster or to import an existing one into Rancher. The steps to take for either method are detailed in the Rancher documentation (See `Cluster Provisioning`).</br>
 *While it is possible (for development) to deploy Epinio on the same node where Rancher is installed, this requires more steps and is not supported by Rancher either.*
 
 ### Rancher settings
+
 At the time of writing, Epinio is not included in the latest stable Rancher version.</br>
 This means that you have to update two settings in your Rancher dashboard.</br>
 Go to `Global settings` > `Advanced settings`  and update the following settings:</br>
 
 | | |
 |--|--|
-| `ui-dashboard-index`  | `https://releases.rancher.com/dashboard/epinio-v0.3.0/index.html` |
+| `ui-dashboard-index`  | `https://releases.rancher.com/dashboard/epinio-v0.6.0/index.html` |
 | `ui-offline-preferred`  | `Remote` |
 
-*Please, keep in mind that `ui-dashboard-index` value will change over the time until it is not merged in stable Rancher version.*
+*Please, keep in mind that `ui-dashboard-index` value will change over the time until it is merged in stable Rancher version.
+
+## Epinio Prerequisites
+
+### Install an Ingress controller
+
+If there is no Ingress controller running yet on the cluster, install Traefik.
```

Comment:
```suggestion
If there's no Ingress controller running yet on the cluster, you need to install one. In the example below, Traefik will be installed.
```

Reply:
The Traefik installation link might need to be updated/standardized if we explain it somewhere else.

Reply:
In that case, I think I can put this link: https://docs.epinio.io/installation/installation.html#ingress-controller